core/include/webview/detail/backends/cocoa_webkit.hh

```diff
@@ -98,41 +96,42 @@ public:
   cocoa_wkwebview_engine &operator=(cocoa_wkwebview_engine &&) = delete;
 
   virtual ~cocoa_wkwebview_engine() {
+    using namespace cocoa;
```

Comment:
Can we `using namespace cocoa;` only once at the top of the namespace declaration? Could save a lot of repetition.

Reply:
It's repetitive but avoids polluting the `detail` namespace with all the new symbols.

The latest changes reduce the amount of `using namespace`s by...
* Encapsulating the code that depend on symbols from`cocoa` and `webkit` namespaces into a new namespace `cocoa_webkit`.
* Pulling symbols from `objc::literals` into `cocoa` and `webkit`. This is OK because the symbols are used everywhere within `cocoa` and `webkit`.